### ISSUE TITLE
numlock's code

### ISSUE THREAD

**Issue by riorivaldi13** [arduino-libraries/Keyboard#22]
hello,

how can i get numlock code? because in the modifier keys of Keyboard.h there isn't code of numlock

thank you.

**Comment by thekunalsaini** [comment]
Numlock code please explain clearly @riorivaldi13

**Comment by panktishah62** [comment]
Do you want to use alternate functions using the Numlock key (like the modifier keys) ? @riorivaldi13

**Comment by edgar-bonet** [comment]
This issue has been fixed on 2021-11-29 (PR #57). Using the master branch, you can now

```c++
Keyboard.write(KEY_NUM_LOCK);
```

Note that the feature is not yet available in a released version of the library.

**Comment by per1234** [comment]
Closing as resolved by https://github.com/arduino-libraries/Keyboard/pull/57

I used this thing as an accessible cross-platform way to verify keypresses:

https://codepen.io/QDeltaE/pen/PjXWZr

Test sketch:

```cpp
#include <Keyboard.h>

const byte enablePin = 2;
const byte LEDPin = LED_BUILTIN;

void setup() {
  pinMode(enablePin, INPUT_PULLUP);
  pinMode(LEDPin, OUTPUT);
  Keyboard.begin();
}

void loop() {
  if (digitalRead(enablePin) == LOW) {
    Keyboard.write(KEY_NUM_LOCK);
    digitalWrite(LEDPin, HIGH);
    delay(500);
    digitalWrite(LEDPin, LOW);
    delay(500);
  }
}
```

"Keyboard Debugger" log:

```text
>Keydown ↓ Key: "NumLock" KeyCode: 144
>Keyup   ↑ Key: "NumLock" KeyCode: 144
>Keydown ↓ Key: "NumLock" KeyCode: 144
>Keyup   ↑ Key: "NumLock" KeyCode: 144
>Keydown ↓ Key: "NumLock" KeyCode: 144
>Keyup   ↑ Key: "NumLock" KeyCode: 144
```

Thanks @edgar-bonet!
